.devcontainer/devcontainer.json

```diff
@@ -0,0 +1,83 @@
+// For format details, see https://aka.ms/devcontainer.json. For config options, see the README at:
+// https://github.com/microsoft/vscode-dev-containerdevcotns/tree/v0.238.0/containers/go
+{
+  "name": "ismrmrd",
+  "build": {
+    "dockerfile": "../Dockerfile",
+    "target": "devcontainer",
+    "context": ".."
+  },
+  "runArgs": ["--cap-add=SYS_PTRACE", "--security-opt", "seccomp=unconfined"],
+
+  // Configure tool-specific properties.
+  "customizations": {
+    // Configure properties specific to VS Code.
+    "vscode": {
+      // Set *default* container specific settings.json values on container create.
+      "settings": {
+        "files.trimFinalNewlines": true,
+        "files.trimTrailingWhitespace": true,
+        "files.insertFinalNewline": true,
+        "go.toolsManagement.checkForUpdates": "local",
+        "go.useLanguageServer": true,
+        "gopls": { "ui.semanticTokens": true },
+
+        "[cpp]": {
+          "editor.formatOnSave": true
```

Comment:
Not sure this is a good idea for this codebase.

Reply:
Removed.